src/octokit/commands/getRepos.ts

```diff
@@ -62,7 +66,7 @@ query getRepos ($after: String) {
   viewer {
     repositories(
       first: 100,
-      affiliations: [OWNER, ORGANIZATION_MEMBER, COLLABORATOR], ownerAffiliations:[OWNER, ORGANIZATION_MEMBER, COLLABORATOR],
+      affiliations: [OWNER], ownerAffiliations:[OWNER],
```

Comment:
Those two fields aren't needed to fetch user repos

Reply:
Actually due to some weird behaviour in the API, without this you get back some org repos, like I think they are ones I created myself in the org